src/standalone/secret.go

```diff
@@ -31,6 +31,22 @@ type SecretConfig struct {
 	ClusterID string `json:"clusterID"`
 }
 
+func (secret SecretConfig) logContent() {
```

Comment:
Conditional secrets printing gives a hint of given field is set or not. 
It should be either never printed or always printed.

Reply:
every field is always printed (the logger does it like that, and I don't want to list it out by hand)
if its not set then its printed with an empty value
if its set to something ==> "***"

Reply:
Nit: add `const secretValue = "***"` and replace hardocded `"***"` with `secretValue`